etna/transforms/sklearn.py

```diff
@@ -1,20 +1,37 @@
+from enum import Enum
 from typing import List
 from typing import Optional
 from typing import Union
 
+import numpy as np
 import pandas as pd
 from sklearn.base import TransformerMixin
 
 from etna.transforms.base import Transform
 
 
+class TransformMode(str, Enum):
+    """Enum for different metric aggregation modes."""
+
+    macro = "macro"
+    per_segment = "per-segment"
+
+    @classmethod
```

Comment:
We should remove this.
Maybe we will add this with mixin later for every enum.

Reply:
Done